### ISSUE TITLE
Not working with Oreo

### ISSUE THREAD

**Issue by SairesArt** [dahlia/muzei-pixiv#22]
Recently updated to Oreo and it seems to Error out when getting new images.
In the Sources tab of muzei it shows the frowny Puzzle.

**Comment by d3m3vilurr** [comment]
I'll check that. but I'm still using nougat and need the time for brave about installing new rom :p

**Comment by SairesArt** [comment]
Glad to assist

**Comment by SairesArt** [comment]
glad to assist with some kind of dump showing what causes the error if you telle.how

**Comment by SairesArt** [comment]
Just reverted back to Nougat due to other problems with Oreo.
Pixiv Muzei is still broken.
I reinstalled cleanly. The app shows the name of the image to be downloaded, but it never downloads and shows just the sad puzzle.

I don't have the play store or any google play services installed. Is this maybe the cause?

PS: Ohh, German Autocorrect destroyed my last posts.

**Comment by d3m3vilurr** [comment]
can you check the storage permission?
anyway, I also sometimes got a configure issue. I'm not using `wifi only` option, but sometimes application thought enabled this option.

**Comment by SairesArt** [comment]
I figured it out.
I could not get any download going, whilst not having an account and logging in with the app.
After I made an account it still didn't work, but I haven't had confirmed said account.

After I confirmed my E-Mail with pixiv it works flawlessly. Maybe a timing coincidence? Interestingly enough the very first time I installed the app, with no pixiv account, it managed to download one image, but failed to ever do a second one.

Works now for me, so no complaints, all great. Great app, thanks for all the waifu goodness.
